### ISSUE TITLE
Add CommonJS support.

### ISSUE THREAD

**Issue by pamelalozano16** [stylelint-stylistic/stylelint-stylistic#54]
### What is the problem you're trying to solve?

Currently, the package only supports ESM, which limits compatibility with environments that rely on CommonJS.

### What solution would you like to see?

Add a CJS version of the package.

**Comment by MorevM** [comment]
Good day [pamelalozano16](https://github.com/pamelalozano16)

Can you explain what scenario you need to run?
The package is shipped as ESM only - that's true, but the actual use case allows configuration via CJS, I mean that

```js
// .stylelintrc.cjs

module.exports = {
  plugins: [
    '@stylistic/stylelint-plugin',
  ],
  rules: {
    '@stylistic/color-hex-case': 'lower',
  },
};
```

works absolutely fine.

So why do you need a dual export?

_* I'm not attacking if you think it sounded aggressive, just don't want to do extra work without a real need :)_

**Comment by firefoxic** [comment]
I agree with the previous comment.

Besides, Stylelint itself is already distributed as ESM-only. We definitely won't be generating CJS.
